src/components/CookieConsentBanner.js

```diff
@@ -38,9 +38,12 @@ const stopGA = () => {
 const CookieConsentBanner = () => {
   const [isBannerOpen, setIsBannerOpen] = React.useState(false)
   const [hasTrackedView, setHasTrackedView] = React.useState(false)
-  const [acceptanceCookie, setAcceptanceCookie] = React.useState(
-    Cookies.get(ACCEPTANCE_COOKIE)
-  )
+  const [acceptanceCookie, setAcceptanceCookie] = React.useState(false)
+
+  // #438: perform check client side to avoid cookie banner briefly appearing
+  React.useEffect(() => {
```

Comment:
My shallow understanding of `useEffect` is that it gives the component lifecycle... so I am guessing this means that it does the stuff in the function after the component has been through them. If so. Seems reasonable.

Reply:
So in this case, the `useEffect` is equivalent to a `componentDidMount`. This lifecycle method does not get called during SSR.